gnmi_server/client_subscribe.go

```diff
@@ -207,6 +208,10 @@ func (c *Client) Run(stream gnmipb.GNMI_SubscribeServer) (err error) {
 	c.Close()
 	// Wait until all child go routines exited
 	c.w.Wait()
+	if strings.Contains(err.Error(), "i/o timeout") {
```

Comment:
>Contains [](http://example.com/codeflow?start=12&length=8)

Match error message string will be fragile? Which library assign this error message? Could you use error code?

Reply:
Right, but seems this will require NOT small changes, like now we're using priorityQueue in all db_client which only stores error messages, refer https://github.com/sonic-net/sonic-gnmi/blob/01fe667e7d44cdfb2d046064c18e4d14ac6c0b4d/sonic_data_client/db_client.go#L833, do you suggest we polish this part now or later together with the project change?